src/server/rdb_save.cc

```diff
@@ -529,6 +540,11 @@ error_code RdbSerializer::SaveStreamObject(const robj* obj) {
   return error_code{};
 }
 
+error_code RdbSerializer::SaveJsonObject(const PrimeValue& pv) {
+  auto json_string = pv.GetJson()->to_string();
+  return SaveString(json_string);
+}
```

Comment:
In the future we should maybe enable one of jsoncons extensions and use a binary format like bson